python/lsst/ip/isr/brighterFatterKernel.py

```diff
@@ -64,8 +64,9 @@ def __init__(self, camera=None, level=None, **kwargs):
         self.level = level
 
         # Things inherited from the PTC
-        self.means = dict()
```

Comment:
Can you file a ticket, and leave a TODO in the class docstring to document what is stored in the BFK calibration?  I was wondering why this change didn't extend to that, but then I realized that I never filled that out when I did this ticket.

Reply:
DM-35260